synapse/server_notices/server_notices_manager.py

```diff
@@ -50,20 +52,21 @@ def is_enabled(self):
         return self._config.server_notices_mxid is not None
 
     async def send_notice(
-        self, user_id, event_content, type=EventTypes.Message, state_key=None
-    ):
+        self,
+        user_id: str,
+        event_content: dict,
+        type: str = EventTypes.Message,
```

Comment:
Why `str` instead of `EventTypes`?

Reply:
`EventTypes` is just an object with a bunch of string properties. `EventTypes.Message` is not an instance of `EventTypes`.

Reply:
aha, fair enough.